easy-ui-tokens/src/theme/base.json

```diff
@@ -33,10 +49,29 @@
     "color.text.disabled": { "value": "{color.gray.300}" },
     "color.text.heading": { "value": "{color.blue.800}" },
     "color.text.inverse": { "value": "{color.gray.000}" },
-    "color.text.success.inverse": { "value": "{color.blue.800}" },
-    "color.text.success.inverse.pressed": { "value": "{color.gray.000}" },
+    "color.text.black.inverse.bold": { "value": "{color.gray.000}" },
+    "color.text.black.inverse.subtle": { "value": "{color.blue.800}" },
+    "color.text.danger.bold": { "value": "{color.red.800}" },
```

Comment:
i added some new text colors, but tbh, idk how i feel about having a token text color name with `color.text.danger.bold`, open to ideas on it though. i thought about maybe creating another set of background tokens, something like `"color.background.<color>.notification"` and the corresponding text would just follow the inverse pattern, but it maybe felt a little too specific lol

Reply:
i think for now this is good. it seems to mirror what other design systems are doing with their alias tokens.

i think we can have a retro on this after we're done with our atoms to see how the patterns aged